docs/essentials/exceptions/exceptions-best-practices.md

```diff
@@ -257,63 +148,6 @@ The following list contains guidelines for creating your own exceptions and when
   End Class
   ```
 
-  C++
-  ```
-  ref class FileReader
-  {
-  private:
-      String^ fileName;
-
-  public:
-      FileReader(String^ path)
-      {
-          fileName = path;
-      }
-
-      array<Byte>^ Read(int bytes)
-      {
-          array<Byte>^ results = FileUtils::ReadFromFile(fileName, bytes);
-          if (results == nullptr)
-          {
-              throw NewFileIOException();
-          }
-          return results;
-      }
-
-      FileReaderException^ NewFileIOException()
-      {
-          String^ description = "My NewFileIOException Description";
-
-          return gcnew FileReaderException(description);
-      }
-  };
-  ```
-
-  Visual Basic
-  ```
-  Class FileReader
-      Private fileName As String
-
-      Public Sub New(path As String)
-          fileName = path
-      End Sub
-
-      Public Function Read(bytes As Integer) As Byte()
-          Dim results() As Byte = FileUtils.ReadFromFile(fileName, bytes)
-          If results Is Nothing
-              Throw NewFileIOException()
-          End If
-          Return results
-      End Function
-
-      Function NewFileIOException() As FileReaderException
-          Dim description As String = "My NewFileIOException Description"
-
-          Return New FileReaderException(description)
-      End Function
-  End Class
-  ```
-
-  Alternatively, use the exception's constructor to build the exception. This is more appropriate for global exception classes such as [ArgumentException](https://msdn.microsoft.com/library/system.argumentexception).
+  Alternatively, use the exception's constructor to build the exception. This is more appropriate for global exception classes such as [ArgumentException](http://dotnet.github.io/api/System.ArgumentException.html).
```

Comment:
The "alternatively" seems like it is reaching for it's context, after a sample. Is there a better way to phrase or structure that content?